holidays/countries/vietnam.py

```diff
@@ -34,29 +39,29 @@ def _populate_public_holidays(self):
         dts_observed = set()
 
         # New Year's Day
-        dts_observed.add(self._add_new_years_day("International New Year's Day"))
+        dts_observed.add(self._add_new_years_day(tr("Tết Dương lịch")))
 
         # Lunar New Year
-        self._add_chinese_new_years_eve("Vietnamese New Year's Eve")
-        self._add_chinese_new_years_day("Vietnamese New Year")
-        self._add_chinese_new_years_day_two("The second day of Tet Holiday")
-        self._add_chinese_new_years_day_three("The third day of Tet Holiday")
-        self._add_chinese_new_years_day_four("The forth day of Tet Holiday")
-        self._add_chinese_new_years_day_five("The fifth day of Tet Holiday")
+        self._add_chinese_new_years_eve(tr("Giao thừa"))
+        self._add_chinese_new_years_day(tr("Tết Nguyên Đán"))
+        self._add_chinese_new_years_day_two(tr("Mùng hai Tết Nguyên Đán"))
+        self._add_chinese_new_years_day_three(tr("Mùng ba Tết Nguyên Đán"))
+        self._add_chinese_new_years_day_four(tr("Mùng bốn Tết Nguyên Đán"))
+        self._add_chinese_new_years_day_five(tr("Mùng năm Tết Nguyên Đán"))
 
         # Vietnamese Kings' Commemoration Day
         # https://en.wikipedia.org/wiki/H%C3%B9ng_Kings%27_Festival
         if self._year >= 2007:
-            dts_observed.add(self._add_hung_kings_day("Hung Kings Commemoration Day"))
+            dts_observed.add(self._add_hung_kings_day(tr("Giỗ Tổ Hùng Vương")))
 
         # Liberation Day/Reunification Day
-        dts_observed.add(self._add_holiday_apr_30("Liberation Day/Reunification Day"))
+        dts_observed.add(self._add_holiday_apr_30(tr("Ngày Chiến thắng")))
 
         # International Labor Day
-        dts_observed.add(self._add_labor_day("International Labor Day"))
+        dts_observed.add(self._add_labor_day(tr("Ngày Quốc tế lao động")))
 
         # Independence Day
-        dts_observed.add(self._add_holiday_sep_2("Independence Day"))
```

Comment:
We might also want to take this opportunity to standardize the `en_US` localization for Vietnam holidays as well, on top of the typo fixes like `forth day` → `fourth day` from the existing VN implementation.

I'm using the [Official English translation of No. 8662/VPCP-KGVX](https://thuvienphapluat.vn/cong-van/EN/Lao-dong-Tien-luong/Official-Dispatch-8662-VPCP-KGVX-2023-Lunar-New-Year-Holiday-National-Day-Holiday-Schedules-2024/585800/tieng-anh.aspx) and [Vietnam Government News](https://en.baochinhphu.vn/viet-nam-to-have-17-public-holiday-dates-in-2024-111231225153729932.htm) as our sources for now:
```python
        # New Year's Day
        dts_observed.add(self._add_new_years_day(tr("Tết Dương lịch")))

        # Lunar New Year's Eve
        self._add_chinese_new_years_eve(tr("Giao thừa"))

        # Lunar New Year
        self._add_chinese_new_years_day(tr("Tết Nguyên Đán"))

        # Second Day of Lunar New Year
        self._add_chinese_new_years_day_two(tr("Mùng hai Tết Nguyên Đán"))

        # Third Day of Lunar New Year
        self._add_chinese_new_years_day_three(tr("Mùng ba Tết Nguyên Đán"))

        # Fourth Day of Lunar New Year
        self._add_chinese_new_years_day_four(tr("Mùng bốn Tết Nguyên Đán"))

        # Fifth Day of Lunar New Year
        self._add_chinese_new_years_day_five(tr("Mùng năm Tết Nguyên Đán"))

        # Hung Kings' Festival
        if self._year >= 2007:
            dts_observed.add(self._add_hung_kings_day(tr("Giỗ Tổ Hùng Vương")))

        # Reunification Day
        dts_observed.add(self._add_holiday_apr_30(tr("Ngày Chiến thắng")))

        # International Labor Day
        dts_observed.add(self._add_labor_day(tr("Ngày Quốc tế lao động")))

        # National Day
        dts_observed.add(self._add_holiday_sep_2(tr("Ngày Quốc khánh")))
```